chain-alerter/src/alerts.rs

```diff
@@ -430,14 +430,15 @@ pub enum AlertKind {
         /// The amount of slots.
         slot_amount: u64,
 
+        /// The number of seconds between the block times of the first and last block in the
+        /// state.
+        time_elapsed: u64,
```

Comment:
`seconds_elapsed` is less ambiguous.
```suggestion
        seconds_elapsed: u64,
```